metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-fail2ban",
-  "version": "4.1.1-rc0",
+  "version": "4.1.3-rc0",
```

Comment:
since the release is an ehnancement, please bump it to 4.2.0 and regenerate the changelog.